kubernetes/coredns.yaml.sed

```diff
@@ -79,7 +79,7 @@ metadata:
     k8s-app: kube-dns
     kubernetes.io/name: "CoreDNS"
 spec:
-  replicas: 2
+  replicas: 1
```

Comment:
I think we should remove the line entirely. Otherwise the issue of add-on manager reconciliation and autoscalers remains.